modules/ROOT/pages/design-add-api-dependency.adoc

```diff
@@ -3,8 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In projects for API specifications or RAML API fragments, you can add dependencies of either API fragments (RAML) or
-rulesets. Because the dependencies are referenced in Anypoint Exchange, you cannot directly edit them in your projects.
+In projects for API specifications or RAML API fragments, you can add dependencies of either API fragments (RAML), OAS Components, JSON schemas, or rulesets. Because the dependencies are referenced in Anypoint Exchange, you cannot directly edit them in your projects.
```

Comment:
I would change to `In projects for API specifications or API fragments`, without the RAML part, since we are clarifying below all types of fragments (including RAML fragments)